src/Toast.js

```diff
@@ -73,6 +79,17 @@ const Toast = React.forwardRef(
     bsPrefix = useBootstrapPrefix('toast');
     const delayRef = useRef(delay);
     const onCloseRef = useRef(onClose);
+    const [exited, setExited] = useState(!show);
+
+    const useAnimation = Transition && animation;
```

Comment:
Out of curiosity, why was this originally named `useAnimation`? seems like `hasAnimation` would be more appropriate here, since it's not a custom hook.